raster/src/test/scala/geotrellis/raster/ArrayTileSpec.scala

```diff
@@ -231,3 +257,42 @@ class ArrayTileSpec extends FunSpec
     }
   }
 }
+
+object ArrayTileSpec {
+  /** BiasedAdd is evaluates `x + NoData` as `x` */
+  case object BiasedAdd extends LocalTileBinaryOp {
```

Comment:
This is added as an example where you'd want the `LocalTileBinaryOp` to control the semantics of `NoData`